packages/wallet-adapter-core/package.json

```diff
@@ -54,8 +54,8 @@
     "buffer": "^6.0.3",
     "eventemitter3": "^4.0.7",
     "tweetnacl": "^1.0.3",
-    "@atomrigslab/aptos-wallet-adapter": "^0.1.20",
-    "@aptos-connect/wallet-adapter-plugin": "^1.0.1"
+    "@atomrigslab/aptos-wallet-adapter": "^0.1.21",
+    "@aptos-connect/wallet-adapter-plugin": "^1.0.0"
```

Comment:
Need to revert back to 1.0.1

Reply:
Sorry about that. I've reverted @aptos-connect/wallet-adapter-plugin back to ^1.0.1 as requested.